google/cloud/speech/cloud_speech.yaml

```diff
@@ -11,17 +11,18 @@ documentation:
     Google Cloud Speech API.
 
 apis:
-- name: google.cloud.speech.v1.Speech
+- name: google.cloud.speech.v1beta1.Speech
+- name: google.longrunning.Operations
```

Comment:
This is going to create a problem in the scenario that a client uses multiple APIs that have long-running operations; the long-running operations class(es) will be duplicated and potentially clash.